src/sidebar/panel.js

```diff
@@ -71,6 +71,7 @@ ClassicEditor.create(document.querySelector('#editor'), {
       });
 
       enableSync.onclick = () => {
+        giveFeedbackButton.style.display = 'none';
```

Comment:
Can you put that inside the enableSyncAction function?

Reply:
Sure as you wish